Dockerfile

```diff
@@ -12,22 +12,21 @@ RUN apt-get update && \
     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9 && \
     apt-get update && \
     apt-get install -y \
+    autoconf \
     build-essential \
     git \
     gsl-bin \
+    lbzip2 \
     libbz2-dev \
     libcurl4-openssl-dev \
     libgsl0-dev \
     liblzma-dev \
-    libmysqlclient-dev \
     libncurses5-dev \
-    libpng-dev \
     libssl-dev \
     libxml2-dev \
-    pigz \
     python3.7 \
```

Comment:
That's a good point. I'm using 16.04 here because that's what gemBS is developed and tested against on their end. I agree ideally a newer Ubuntu version would be better though.